llvm/test/CodeGen/AMDGPU/global_atomic_optimizer_fp_rtn.ll

```diff
@@ -1,1170 +1,1623 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
-; RUN:  opt -S -mtriple=amdgcn-- -mcpu=gfx906 -amdgpu-atomic-optimizer-strategy=Iterative -passes='amdgpu-atomic-optimizer,verify<domtree>' %s | FileCheck -check-prefix=IR-ITERATIVE %s
-; RUN:  opt -S -mtriple=amdgcn-- -mcpu=gfx906 -amdgpu-atomic-optimizer-strategy=DPP -passes='amdgpu-atomic-optimizer,verify<domtree>' %s | FileCheck -check-prefix=IR-DPP %s
+; RUN:  opt -S -mtriple=amdgcn-- -mcpu=gfx906 -amdgpu-atomic-optimizer-strategy=Iterative -passes='amdgpu-atomic-optimizer,verify<domtree>' %s | FileCheck --check-prefixes=UNIFORM-VAL,DIV-VAL-ITERATIVE %s
```

Comment:
UNIFORM/DIV are not good check prefixes to use. It only makes sense if you're totally sure about the content of the test changes. It's better if they are descriptive of just the run itself. It's also better to not change the base checks in the change commit. I would suggest keeping IR-ITERATIVE/IR-DPP, and just adding "IR" as the common prefix